qemu/tests/cfg/smbios_default_check.cfg

```diff
@@ -0,0 +1,24 @@
+- smbios_default_check: install setup image_copy unattended_install.cdrom
+    only q35
+    no Host_RHEL.m6,Host_RHEL.m7,Host_RHEL.m8.u0,Host_RHEL.m8.u1
+    # dmidecode not available (and no plans for it) on s390
+    no s390x
+    type = smbios_default_check
+    requires_root = yes
+    start_vm = yes
+    kill_vm = yes
+    # Expected output
+    System_Manufacturer = "^Red Hat$"
+    System_SKU_Number = "^[8,9].[2,3,4,5,6,7,8,9].0$"
```

Comment:
no commas here

Reply:
Thanks for your reminder,i had corrected the problem